internal/ethapi/api_test.go

```diff
@@ -323,7 +326,16 @@ func (b testBackend) StateAndHeaderByNumberOrHash(ctx context.Context, blockNrOr
 }
 func (b testBackend) PendingBlockAndReceipts() (*types.Block, types.Receipts) { panic("implement me") }
 func (b testBackend) GetReceipts(ctx context.Context, hash common.Hash) (types.Receipts, error) {
-	panic("implement me")
+	number := rawdb.ReadHeaderNumber(b.db, hash)
+	if number == nil {
+		return nil, nil
+	}
+	header, err := b.HeaderByNumber(ctx, rpc.BlockNumber(*number))
```

Comment:
What is the need for this check?

Reply:
Seems this check is redundant.

btw, I've rewritten those lines into `HeaderByHash`(-4 cloc), and in that we need to check the header nullable